docs/resources/sonarqube_alm_gitlab.md

```diff
@@ -0,0 +1,33 @@
+# sonarqube_alm_gitlab
+
+Provides a Sonarqube GitLab Alm/Devops Platform Integration resource. This can be used to create and manage a Alm/Devops
+Platform Integration for GitLab.
+
+## Example: Create a GitHub Alm Integration
+
+```terraform
+resource "sonarqube_alm_gitlab" "gitlab-alm" {
+  key                    = "myalm"
+  personal_access_token  = "my_personal_access_token"
+  url                    = "https://gitlab.com/api/v4"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- key - (Required) - Unique key of the GitLab instance setting. Maximum length: 200
+- personal_access_token - (Required) - GitLab App personal access token with
```

Comment:
formatting here looks weird. should be a line per argument. Don't worry about length of that line

Reply:
Oh whoops. I remembered doing that and thinking the same thing. Shoulda just fixed it immediately 😆